projects/go-lib/src/lib/components/go-tab/go-tab-group.component.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../../styles/variables';
+
+.go-tab__header {
+  border-radius: $global-radius $global-radius 0 0;
+  display: inline-flex;
+  font-size: 1.125rem;
+  font-weight: normal;
+  padding: .5rem 1rem;
+
+  &:hover {
+    background-color: $theme-light-bg-hover;
+    cursor: pointer;
```

Comment:
Let's apply this to the `go-tab__header`. Makes for a slightly smoother experience.